### PR TITLE
OCPBUGS-7335: Updates SDK extensions to current

### PR DESCRIPTION
**Pull request by opayne1**
Updates SDK extensions to current

Version(s):
CP to 4.12 and 4.13

Issue:
https://issues.redhat.com/browse/OCPBUGS-7335

Link to docs preview:
https://55844--docspreview.netlify.app/openshift-enterprise/latest/web_console/dynamic-plug-in/dynamic-plug-ins-reference.html#dynamic-plug-in-sdk-extensions_dynamic-plug-ins-reference

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by opayne1** [comment]
@spadgett @jhadvig I think I captured all of the description updates. Let me know if I missed anything. Thanks!

File: modules/dynamic-plug-in-sdk-extensions.adoc [hunk 1]
```diff
@@ -1025,14 +1095,14 @@ events
 [discrete]
 == `console.topology/adapter/build`
 
-`BuildAdapter` contributes an adapter to adapt element to data that can be used by the `Build` component.
+`BuildAdapter` contributes an adapter to adapt element to data that can be used by the Pod component.
```
Comment (spadgett) [inline modules/dynamic-plug-in-sdk-extensions.adoc:1098]: Is this change correct?

File: modules/dynamic-plug-in-sdk-extensions.adoc [hunk 2]
```diff
@@ -1025,14 +1095,14 @@ events
...
 
 [cols=",,,",options="header",]
 |===
 |Name |Value Type |Optional |Description
 |`adapt`
 |`CodeRef<(element: GraphElement) => AdapterDataType<BuildConfigData> | undefined>`
-|no |
+|no | adapter to adapt element to data that can be used by Networking component. |
```
Comment (spadgett) [inline modules/dynamic-plug-in-sdk-extensions.adoc:1105]: Is this correct?

File: modules/dynamic-plug-in-sdk-extensions.adoc (hunk 1 above)
Comment (opayne1) [reply modules/dynamic-plug-in-sdk-extensions.adoc:1098]: So like this?
```suggestion
`BuildAdapter` contributes an adapter to adapt element to data that can be used by the Build configuration.
```
and same for the one below?

**Comment by opayne1** [comment]
Closing this PR and replacing with  #57513.

**Pull request closed without merging**